plaso/parsers/cookie_plugins/ganalytics.py

```diff
@@ -355,7 +355,7 @@ def GetEntries(
 
     Args:
       parser_mediator (ParserMediator): parser mediator.
-      cookie_data (bytes): cookie data.
+      cookie_data (str): cookie data.
```

Comment:
if cookie_data is str why do we need to decode it in this function?

Reply:
See comment below.